CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+## This defines the code owner of the repository
+
+@lyonlai @alecthomas
```

Comment:
Should we add some more people here? @jvmakine and @juliaogris perhaps?

Reply:
yup. for sure